code/modules/food_and_drink/sandwiches.dm

```diff
@@ -243,14 +266,71 @@
 		..()
 		reagents.add_reagent("love", 15)
 
+/obj/item/reagent_containers/food/snacks/burger/heartburger/synth
+	name = "heartburger"
+	desc = "A hearty meal, made with Love. This one seems to contain a green synthetic heart."
+	icon_state = "synthheartburger"
+	food_effects = list("food_sweaty_big", "food_hp_up_big")
+	meal_time_flags = MEAL_TIME_LUNCH | MEAL_TIME_DINNER
+
+	New()
+		..()
+		reagents.add_reagent("love", 15)
```

Comment:
You can remove anything from these child types that's the same on the parent, that's the reason for making them child types in the first place!

Reply:
did it, i also fixed a bug where making a flockheart/brainburger would result in a regular flockburger, which now makes the correct burger